lib/workers/repository/package-files.ts

```diff
@@ -46,12 +46,24 @@ export class PackageFiles {
         deps += pad ? '</blockquote>\n</details>\n\n' : '';
         continue;
       }
+
+      const placeHolder = `no version found`;
+      let ver = '';
+      let digest = '';
+      let version = '';
```

Comment:
Why define these at this top level? Aren't they applicable only to the dep scope only?

Reply:
micro-optimizations, I guess it is neglectable in our application.